Frameworks/CoreFoundationAdditions/_NSCFTemporaryRootObject.mm

```diff
@@ -16,4 +16,8 @@
 #include "_NSCFTemporaryRootObject.h"
 
 @implementation _NSCFTemporaryRootObject
+- (void)doesNotRecognizeSelector:(SEL)selector {
+    // According to the reference platform documentation, this method must never return.
+    abort();
```

Comment:
I decided to use `RaiseFailFastException`. Verified that it generates a dump via WER when Local Dumps are enabled. The exception record is intact, too!

---
In reply to: [71582230](https://github.com/Microsoft/WinObjC/pull/671#discussion_r71582230) [](ancestors = 71582230,71576020,71575314,71574896,71574472)